app/bundles/CoreBundle/Tests/DependencyInjection/Compiler/SmsTransportPassTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/*
+ * @copyright   2018 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ * @author      Jan Kozak <galvani78@gmail.com>
```

Comment:
The authorship of each line is stored in git. Let's remove it from annotation. No class in Mautic has this. (in all files)